include/cantera/thermo/BinarySolutionTabulatedThermo.h

```diff
@@ -156,15 +156,15 @@ class BinarySolutionTabulatedThermo : public IdealSolidSolnPhase
     size_t m_kk_tab;
 
     //! Current tabulated species mole fraction
-    double m_xlast;
+    mutable double m_xlast;
 
     //! Vector for storing tabulated thermo
     vector_fp m_molefrac_tab;
     vector_fp m_enthalpy_tab;
     vector_fp m_entropy_tab;
 
 private:
```

Comment:
Just for my own growth, can you explain what adding `const` after the method name does?

Reply:
A method with `const` at the end of the declaration cannot modify any member variables of the class, except those that are marked as `mutable`. See [this SO answer](https://stackoverflow.com/a/3141107/310477) for a more detailed explanation.

The fact that a non-`const` function does not override a `const` function which otherwise has the same signature is one of C++'s many subtle "gotchas".